modules/nodes-nodes-managing-about.adoc

```diff
@@ -5,7 +5,12 @@
 [id="nodes-nodes-managing-about_{context}"]
 = Modifying nodes
 
-To make configuration changes to a cluster, or machine pool, you must create a custom resource definition (CRD), or `KubeletConfig` object. {product-title} uses the Machine Config Controller to watch for changes introduced through the CRD to apply the changes to the cluster.
+To make configuration changes to a cluster, or machine pool, you must create a custom resource definition (CRD), or `kubeletConfig` object. {product-title} uses the Machine Config Controller to watch for changes introduced through the CRD to apply the changes to the cluster.
+
+[NOTE]
+====
+Because the fields in a `kubeletconfig` object are passed directly to the kubelet from upstream Kubernetes, the validation of those fields is handled directly by the kubelet itself. Please refer to the relevant Kubernetes documentation for the valid values for these fields. Invalid values in the `kubeletConfig` object can render cluster nodes unusable.
```

Comment:
```suggestion
Because the fields in a `kubeletConfig` object are passed directly to the kubelet from upstream Kubernetes, the validation of those fields is handled directly by the kubelet itself. Please refer to the relevant Kubernetes documentation for the valid values for these fields. Invalid values in the `kubeletConfig` object can render cluster nodes unusable.
```